ludusavi.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=Ludusavi
+Comment=Tool for backing up your PC video game save data.
+Exec=ludusavi %U
+Icon=ludusavi
```

Comment:
You could as well set it to the right, and name the file properly. And have it that way upstream.

Reply:
outside of flatpaks its common to name  the icon APPNAME.png and my idea would be put it upstream (might only be possible once the next release comes)

Reply:
I prefer to keep the desktop in sync with upstream (We now have a plan for moving the files soon, all but the yaml will become part of upstream)
when @mtkennerly is willing to change the upstream naming scheme for the future, I will gladly change the icon name in the desktop.

Reply:
if it is upstream then why is it here?